elasticsearch.md

```
@@ -2723,6 +2821,186 @@ readonlyrest:
+    users:
+      user_id_attribute: "CN"
+    groups:
+      group_id_attribute: "OU"
+```
+
+The same three modes are available. Where a username takes the first value found, groups take all of
+them — so one `subject_dn_pattern` serves both, and `pattern: "OU=grp-([^,]+)"` yields a group per match.
+
+Note that a DN's OUs are frequently a position in an org tree rather than a set of roles:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)